updates/pgbouncer_fdw--1.1.0--1.2.0.sql

```diff
@@ -0,0 +1,1353 @@
+CREATE TEMP TABLE pgbouncer_fdw_preserve_privs_temp (statement text);
+
+INSERT INTO pgbouncer_fdw_preserve_privs_temp
+SELECT 'GRANT '||string_agg(privilege_type, ',')||' ON @extschema@.pgbouncer_clients TO '||grantee::text||';'
+FROM information_schema.table_privileges
+WHERE table_schema = '@extschema@'
+AND table_name = 'pgbouncer_clients'
```

Comment:
Lesson learned elsewhere 
```suggestion
AND table_name = 'pgbouncer_clients'
AND grantee != 'PUBLIC'
```
Lesson learned elsewhere otherwise you run into this in some scenarios
```
postgres=# alter extension pgbouncer_fdw update;
ERROR:  role "PUBLIC" does not exist
CONTEXT:  SQL statement "GRANT EXECUTE ON FUNCTION public.pgbouncer_clients_func()  TO "PUBLIC","postgres";"
PL/pgSQL function inline_code_block line 7 at EXECUTE
```